presto-tests/src/main/java/com/facebook/presto/tests/AbstractTestAggregations.java

```diff
@@ -754,6 +754,19 @@ public void testTotalValuesSizeInBytes()
         assertQuery("SELECT total_size_in_bytes(ROW(comment, comment)) FROM orders", "SELECT 2 * 727364");
     }
 
+    @Test
+    public void testMaxInMemorySize()
```

Comment:
why not test array/map/row here?

Reply:
The `$internal$max_data_size_for_stats` is needed for the opensource only. The opensource metastore doesn't store the statistics for complex types.